src/Plugin/Field/FieldFormatter/EntityReferenceMenuFormatter.php

```diff
@@ -0,0 +1,153 @@
+<?php
+namespace Drupal\y_lb\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Block\BlockManagerInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceFormatterBase;
+use Drupal\Core\Render\RendererInterface;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Plugin implementation of the 'entity reference rendered menu' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "entity_reference_menu_view",
+ *   label = @Translation("Rendered menu"),
+ *   description = @Translation("Display the referenced menu"),
+ *   field_types = {
+ *     "entity_reference"
+ *   }
+ * )
+ */
+class EntityReferenceMenuFormatter extends EntityReferenceFormatterBase {
```

Comment:
this looks like a Formatter to be placed into menu related module, maybe in inline_entity_form as a submodule or openy_er module, where we do have Entity Reference Enxhancements

Reply:
I agree, that this also should go to the IEF contrib submosule

Reply:
this one https://git.drupalcode.org/project/inline_entity_form/-/tree/2.0.x/modules/inline_entity_menu_form @aleevas

Reply:
@anpolimus @podarok Here is the MR to the IEF module https://git.drupalcode.org/project/inline_entity_form/-/merge_requests/85